framework/core/less/forum/UserSecurityPage.less

```diff
@@ -0,0 +1,80 @@
+.UserSecurityPage {
+  > ul, fieldset > ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  > ul > li {
+    margin-bottom: 25px;
+  }
+}
+
+.AccessTokensList {
+  display: flex;
+  flex-direction: column;
+  gap: 1px;
```

Comment:
Flex gap isn't supported on iOS 14.0 and below. We should probably use:

```css
> *:not(:first-child) {
  margin-left: 1px;
}
```

Reply:
why must you destroy my hopes and dreams

Reply:
just sharing the safari love 🥰🥰